.env.example.docker

```diff
@@ -0,0 +1,56 @@
+APP_ENV=production
+APP_KEY=base64:FbQWGGyjpWBqY4QwuiLySgcs4SuiRKZAvt3450NX/z8=
+APP_DEBUG=false
+APP_LOG_LEVEL=error
+APP_URL=http://login-module.test
+
+DB_CONNECTION=mysql
+DB_HOST=mysql
+DB_PORT=3306
+DB_DATABASE=login_module
+DB_USERNAME=root
+DB_PASSWORD=buddy
+DB_CHARACTER_SET=utf8
+
+BROADCAST_DRIVER=log
+CACHE_DRIVER=file
+SESSION_DRIVER=file
+QUEUE_DRIVER=sync
+
+REDIS_HOST=redis
+REDIS_PASSWORD=null
+REDIS_PORT=6379
+
+MAIL_DRIVER=smtp
+MAIL_HOST=mailhog
+MAIL_PORT=1025
+MAIL_USERNAME=null
+MAIL_PASSWORD=null
+MAIL_ENCRYPTION=null
+MAIL_FROM_ADDRESS=info@france-ioi.org
+
+PUSHER_APP_ID=
+PUSHER_APP_KEY=
+PUSHER_APP_SECRET=
+
+FACEBOOK_ID=1612502442389113
```

Comment:
I know those (expired) credentials were present in the original `.env.example` but it's a good moment to think about removing them (from both files really).

Reply:
fixed